src/Components/StepThree/StepThree.js

```diff
@@ -1,112 +1,294 @@
+                        onChange={handleChangeRadio}/>
+                <label htmlFor="doesHaveAgricultureSkillsYes">Yes</label>
                 </div>
                 <div>
-                <input type="radio" id="skillsAgrNo" value="No" name="skAgr"/>
-                <label for="skillsAgrNo">No</label>
+                <Input 
+                        type={"radio"}
+                        id={"doesHaveAgricultureSkillsNo"}
+                        value={""} 
+                        name={"doesHaveAgricultureSkills"} 
+                        onChange={handleChangeRadio}/>
+                <label htmlFor="doesHaveAgricultureSkillsNo">No</label>
                 </div>
             </div>
         </div>
-
-        <div className="skillsDescription">
+        {inputs.doesHaveAgricultureSkills && <div className="skillsDescription">
             <p><span>*</span> What? Please describe</p>
             <div>
-                <textarea type="text" rows="6" cols="100"/>
+                <textarea 
+                type="text" 
+                rows="6" 
+                cols="80" 
+                name="agricultureSkills"
+                value={inputs.agricultureSkills}
+                onChange={handleChange}/>
             </div>
-        </div>
+        </div>}
 
         <div className="skills">
             <div className="metalWorkIf">
             <p><span>*</span> Do you have any metalwork skills?</p>
             <div>
                 <div>
-                <input type="radio" id="skillsMetalWorkYes" value="yes" name="skMW"/>
-                <label for="skillsMetalWorkYes">Yes</label>
+                <Input 
+                        type={"radio"}
+                        id={"skillsMetalWorkYes"}
+                        value={true} 
+                        name={"doesHaveMetalworkSkills"} 
+                        onChange={handleChangeRadio}/>
+                <label htmlFor="skillsMetalWorkYes">Yes</label>
                 </div>
                 <div>
-                <input type="radio" id="skillsMetalWorkNo" value="No" name="skMW"/>
-                <label for="skillsMetalWorkNo">No</label>
+                <Input 
+                        type={"radio"}
+                        id={"skillsMetalWorkNo"}
+                        value={""} 
+                        name={"doesHaveMetalworkSkills"} 
+                        onChange={handleChangeRadio}/>
+                <label htmlFor="skillsMetalWorkNo">No</label>
                 </div>
             </div>
             </div>
             </div>
-
-        <div className="metalWorkWhat">
+        {inputs.doesHaveMetalworkSkills && <div className="metalWorkWhat">
             <p><span>*</span> What? Please select all that apply</p>
             <div className="selectMetalWork">
                     <div>
-                    <input type="checkbox" id="marking" name="metalWork"/>
-                    <label for="marking">Marking</label>
+                    <Input 
```

Comment:
This is a list of inputs and they are basically all the same - only value (and id) are different. This is a good place to use map, and display the inputs dynamically. 
We can have an array of metalwork skills: [marking, cutting, drilling, blah, blah ... ] and then map that array to Input components.

Reply:
done